core/agents/troubleshooter.py

```diff
@@ -182,6 +182,7 @@ async def get_user_feedback(
             return False, False, ""
 
         if user_response.button == "loop":
+            await telemetry.trace_code_event("stuck-in-loop", {"clicked": True})
```

Comment:
add:
- number of epics (is it the initial app or feature?)
- number of all tasks (is it the last task? Could be that we have most iterations on the last task because everyone clicks "continue" until it is the last task)
- current task index
- some data so we can find and analyze the db
- number of iterations
- number of steps
- tech stack? Then we can see easily on which stack is this button used the most